tests/integ/test_tf_efs_fsx.py

```diff
@@ -170,10 +169,10 @@ def test_tuning_tf_script_mode_efs(efs_fsx_setup, sagemaker_session, cpu_instanc
     assert best_training_job
 
 
-@pytest.mark.skipif(
-    tests.integ.test_region() not in tests.integ.EFS_TEST_ENABLED_REGION,
-    reason="EFS integration tests need to be fixed before running in all regions.",
-)
+# @pytest.mark.skipif(
+#     tests.integ.test_region() not in tests.integ.EFS_TEST_ENABLED_REGION,
+#     reason="EFS integration tests need to be fixed before running in all regions.",
+# )
```

Comment:
If EFS tests are fixed by now, we can just remove the skipif lines.

Reply:
It was passed tested when canary (us-east-1) was deployed to my account.